<?xml version="1.0" encoding="UTF-8"?><ui version="4.0"> <class>DeviceControlMainPage</class> <widget class="QWidget" name="DeviceControlMainPage">  <property name="geometry">   <rect>    <x>0</x>    <y>0</y>    <width>1500</width>    <height>779</height>   </rect>  </property>  <property name="minimumSize">   <size>    <width>1330</width>    <height>0</height>   </size>  </property>  <property name="windowTitle">   <string>Form</string>  </property>  <property name="styleSheet">   <string notr="true"/>  </property>  <layout class="QVBoxLayout" name="verticalLayout">   <item>    <widget class="QWidget" name="widget_head" native="true">     <property name="sizePolicy">      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">       <horstretch>0</horstretch>       <verstretch>0</verstretch>      </sizepolicy>     </property>     <property name="minimumSize">      <size>       <width>0</width>       <height>80</height>      </size>     </property>     <property name="maximumSize">      <size>       <width>16777215</width>       <height>80</height>      </size>     </property>     <layout class="QHBoxLayout" name="horizontalLayout">      <item>       <widget class="QPushButton" name="pushButton_start">        <property name="minimumSize">         <size>          <width>100</width>          <height>32</height>         </size>        </property>        <property name="maximumSize">         <size>          <width>200</width>          <height>16777215</height>         </size>        </property>        <property name="font">         <font>          <pointsize>12</pointsize>         </font>        </property>        <property name="styleSheet">         <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>        </property>        <property name="text">         <string>start</string>        </property>       </widget>      </item>      <item>       <widget class="QPushButton" name="pushButton_stop">        <property name="enabled">         <bool>false</bool>        </property>        <property name="minimumSize">         <size>          <width>100</width>          <height>32</height>         </size>        </property>        <property name="maximumSize">         <size>          <width>200</width>          <height>16777215</height>         </size>        </property>        <property name="font">         <font>          <pointsize>12</pointsize>         </font>        </property>        <property name="styleSheet">         <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>        </property>        <property name="text">         <string>stop</string>        </property>       </widget>      </item>      <item>       <widget class="QPushButton" name="pushButton_reset">        <property name="minimumSize">         <size>          <width>100</width>          <height>32</height>         </size>        </property>        <property name="maximumSize">         <size>          <width>200</width>          <height>16777215</height>         </size>        </property>        <property name="font">         <font>          <pointsize>12</pointsize>         </font>        </property>        <property name="styleSheet">         <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>        </property>        <property name="text">         <string>reset</string>        </property>       </widget>      </item>      <item>       <widget class="QPushButton" name="pushButton_checknew">        <property name="minimumSize">         <size>          <width>150</width>          <height>32</height>         </size>        </property>        <property name="maximumSize">         <size>          <width>200</width>          <height>16777215</height>         </size>        </property>        <property name="font">         <font>          <pointsize>12</pointsize>         </font>        </property>        <property name="styleSheet">         <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>        </property>        <property name="text">         <string>check New</string>        </property>       </widget>      </item>      <item>       <widget class="QPushButton" name="pushButton_test">        <property name="minimumSize">         <size>          <width>100</width>          <height>32</height>         </size>        </property>        <property name="maximumSize">         <size>          <width>200</width>          <height>16777215</height>         </size>        </property>        <property name="font">         <font>          <pointsize>12</pointsize>         </font>        </property>        <property name="styleSheet">         <string notr="true">QPushButton{border-radius: 4px;background-color: rgb(50, 157, 253);color: white;}QPushButton:hover{background-color: rgb(29, 148, 255);}QPushButton:pressed{background-color: rgb(10, 139, 255);}QPushButton:disabled{background-color: lightgray;}</string>        </property>        <property name="text">         <string>Test</string>        </property>       </widget>      </item>      <item>       <spacer name="horizontalSpacer">        <property name="orientation">         <enum>Qt::Horizontal</enum>        </property>        <property name="sizeHint" stdset="0">         <size>          <width>40</width>          <height>20</height>         </size>        </property>       </spacer>      </item>     </layout>    </widget>   </item>   <item>    <widget class="QListWidget" name="listWidget">     <property name="sizePolicy">      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">       <horstretch>0</horstretch>       <verstretch>0</verstretch>      </sizepolicy>     </property>     <property name="minimumSize">      <size>       <width>1350</width>       <height>600</height>      </size>     </property>     <property name="alternatingRowColors">      <bool>true</bool>     </property>    </widget>   </item>   <item>    <widget class="QWidget" name="widget" native="true"/>   </item>  </layout> </widget> <resources/> <connections/></ui>